e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -457,9 +457,17 @@ describe("scenarios > embedding-sdk > interactive-question", () => {
 
     getSdkRoot().within(() => {
       cy.findByText(`id = ${FIRST_COLLECTION_ENTITY_ID}`).should("exist");
+
+      cy.log("click on the button to switch target collection");
       cy.findByText("use second collection").click();
       cy.findByText(`id = ${SECOND_COLLECTION_ENTITY_ID}`).should("exist");
 
+      cy.log("close any existing open popovers to reduce flakes");
+      cy.findByText("Data").click();
+
+      cy.log("open the data picker");
+      cy.findByText("Pick your starting data").click();
```

Comment:
We clicked on `use second collection` previously which closes the data picker's popover. We need to re-trigger it.